test/integration/clean/test_clean.py

```diff
@@ -0,0 +1,157 @@
+import os
+import re
+
+import pytest
+import ttapp
+
+import utils
+
+tarantool_major_version, tarantool_minor_version = utils.get_tarantool_version()
+
+skip_cluster_cond = tarantool_major_version < 3
+skip_cluster_reason = "skip cluster instances test for Tarantool < 3"
```

Comment:
We already have this condition in function in `test/utils.py`:

https://github.com/tarantool/tt/blob/6a9c221cbac8f909a17e1595498b79ff6e33b1fd/test/utils.py#L552-L554

Reply:
It seems I just copy-pasted the wrong approach. I'll switch to this one.

Reply:
Done